production.Dockerfile

```diff
@@ -35,8 +35,7 @@ RUN apk --update --no-cache add \
     "make~=4.3" \
     "g++~=10.3" \
     "gcc~=10.3" \
-    "python3~=3.9" \
```

Comment:
Not part of this PR but I thought we were using Python `3.8.12` (see tests, docs, etc..)